specs/altair/validator.md

```diff
@@ -187,25 +189,33 @@ After constructing the `BeaconBlockBody` as per that section, the proposer has a
 
 ##### Sync committee
 
-The proposer receives a number of `SyncCommitteeContribution`s (wrapped in `SignedContributionAndProof`s on the wire) from validators in the sync committee who are selected to partially aggregate signatures from the full committee (see below for details).
+The proposer receives a number of `SyncCommitteeContribution`s (wrapped in `SignedContributionAndProof`s on the wire) from validators in the sync committee who are selected to partially aggregate signatures from individual subcommittees formed by breaking the full sync committee into `SYNC_COMMITTEE_SUBNET_COUNT` pieces (see below for details).
 
 The proposer collects these contributions for further aggregation when preparing a block. 
+Proposers should select the best contribution seen across all aggregators for a given subnet/subcommittee when preparing a block. 
+A contribution with more valid signatures is better than a contribution with fewer signatures.
+
 Recall `block.body.sync_aggregate.sync_committee_bits` is a `Bitvector` where the `i`th bit is `True` if the corresponding validator in the sync committee has produced a valid signature, and that `block.body.sync_aggregate.sync_committee_signature` is the aggregate BLS signature combining all of the valid signatures.
 
-Given a collection of `contributions` and the `BeaconBlock` under construction, the proposer processes them as follows:
+Given a collection of the best seen `contributions` (with no repeating `subcommittee_index` values) and the `BeaconBlock` under construction, the proposer processes them as follows:
 
 ```python
 def process_sync_committee_contributions(state: BeaconState, 
                                          block: BeaconBlock, 
                                          contributions: Set[SyncCommitteeContribution]) -> None:
     sync_aggregate = SyncAggregate()
     signatures = []
+
     for contribution in contributions:
+        subcommittee_index = contribution.subcommittee_index
         for index, participated in enumerate(contribution.aggregate_bits):
             if participated:
-                sync_aggregate.sync_committee_bits[index] = True
+                participant_index = SYNC_COMMITTEE_SUBNET_COUNT * subcommittee_index + index
```

Comment:
I think `SYNC_COMMITTEE_SUBNET_COUNT` is incorrect. Shouldn't this multiple by `size_of_subcommittees` rather than number of subcommittees

Reply:
nice catch